demos/CPP_playVideoFramesDemo.m

```diff
@@ -0,0 +1,60 @@
+% (C) Copyright 2021 CPP_PTB developers
+
+%% Demo showing how to use the createFramesTextureStructure and playVideoFrames functions
+
+% This small script shows how to use the getReponse function
+
+% add parent directory to matlab path (so we can access the CPP_PTB functions)
+addpath(fullfile(pwd, '..'));
+
+%% start with a clean slate
+clear;
+clc;
+
+
+%% initialize PTB % This can def be improved by using other CPP_PTB functions%
+
+    % Skip the PTB sync test
+    Screen('Preference', 'SkipSyncTests', 2);
+
+    % Get the screen numbers and draw to the external screen if avaliable
+    cfg.screen.idx = max(Screen('Screens'));
+
+    % Set the PTB window background manually
+    cfg.color.background = [127 127 127];
+
+    % Get the screen numbers and draw to the external screen if avaliable
+    cfg.screen.idx = max(Screen('Screens'));
+
+    % Open an on screen window
+    [cfg.screen.win, cfg.screen.winRect] = Screen('OpenWindow', cfg.screen.idx, cfg.color.background);
+    
+    
+%% Video related info
+
+    % The name of your "video" and its images format
+    cfg.video.name = 'coffee';
```

Comment:
Do you think that you could add a second video in this demo and have 2 trials ?

I think this would help us how to figure out how to make the code more general